stanford/bin/send-email.py

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env python
 from email.mime.text import MIMEText
+import os
 from subprocess import call
 import sys
 
 def send(recipient, sender, sender_name, subject, body):
-    with open('configuration/stanford/bin/email_params.txt', 'rt') as fin:
+    email_params_file = 'configuration-secure/jenkins/cut-release-branch/email_params.txt'
+    email_params_file = os.environ.get('CONFIGURATION_EMAIL_PARAMS', email_params_file)
```

Comment:
Do we need both these lines?

I don't see CONFIGURATION_EMAIL_PARAMS defined anywhere.

Reply:
It's not defined anywhere. It's meant to be defined to override the defaults.

Reply:
Can you add a comment to the effect that CONFIGURATION_EMAIL_PARAMS is not set on lagunita and under what circumstances it would be set.

Reply:
> If someone decided to set CONFIGURATION_EMAIL_PARAMS where would it be set and which line of code would read it when cut_release_branch job is run from Jenkins? I'm not seeing that piece.

@dcadams This is the bit that reads in the `ENV` var. No manual "reading" is needed; `os.environ` automatically accesses any/all environmental variables [1].

Technically, the var can be set in a number of ways, as long as its done prior to executing the `send-email.py` script.

The most straightforward approach would be like:

```sh
CONFIGURATION_EMAIL_PARAMS='path/to/file.txt' ./send-email.py
```

You could also:

```sh
export CONFIGURATION_EMAIL_PARAMS='path/to/file.txt'
./send-email.py
```

Or when integrating w/ Jenkins, you could even set it as a build option (jenkins build vars get automatically converted into environment variables.


- [1] https://docs.python.org/3/library/os.html#os.environ